htdocs/commande/list_det.php

```diff
@@ -1524,7 +1525,7 @@
 		if (!empty($arrayfields['pr.desc']['checked'])) {
 			// print '<td class="nowrap tdoverflowmax200">'.$obj->description.'</td>';
 			!empty($obj->product_label) ? $labelproduct = $obj->product_label : $labelproduct = $obj->description;
-			print '<td class="nowrap tdoverflowmax200">'.dol_escape_htmltag($labelproduct).'</td>';
+			print '<td class="nowrap tdoverflowmax200">'.dol_string_nohtmltag($labelproduct).'</td>';
```

Comment:
Why dol_string_nohtmltag instead of dol_escape_htmltags ?

Reply:
If free ligne print description

Result if dol_escape_htmltag

![Capture d'écran 2023-10-09 202844](https://github.com/Dolibarr/dolibarr/assets/34568357/c5b6f5fe-dc9d-4ee3-b8ca-3da5272b5c3d)


Result if dol_string_nohtmltag

![Capture d'écran 2023-10-09 203025](https://github.com/Dolibarr/dolibarr/assets/34568357/d0127cb8-08a6-4e12-b2f1-91084e3fd00d)